### PR TITLE
Update cnn.ipynb

### PR DESCRIPTION
**Pull request by ymodak**
fixes https://github.com/tensorflow/tensorflow/issues/45801

### CODE REVIEW

**Comment by lamberta** [comment]
We don't use softmax and prefer logits. Can you keep the code the same update the text instead? Thanks

**Comment by ymodak** [comment]
done

**Review by lamberta — APPROVED** [review]
Thanks!
